src/h5web/explorer/Explorer.tsx

```diff
@@ -22,11 +23,37 @@ function Explorer(props: Props): JSX.Element {
   const domain = useDomain();
   const tree = useMetadataTree();
 
+  const [expandedNodes, setExpandedNodes] = useState<ExpandedNodes>({});
```

Comment:
I moved the `expandedNodes` state, which used to be in `TreeView`, up in the `Explorer`. So it now manages the expanded state of every node in the tree. This allows nodes on the default path to be expanded all at once, and only once, as soon as the tree has been retrieved (cf. `useEffect` below).